views/partials/_whats-new.njk

```diff
@@ -0,0 +1,15 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<div class="app-whats-new" style="background-color:#f8f8f8; border-bottom:1px solid #b1b4b6">
```

Comment:
Are you happy for this styling to be moved to the [main stylesheet](https://github.com/alphagov/govuk-design-system/blob/main/src/stylesheets/main.scss)?

Reply:
Yes I wanted to add it to a stylesheet but didn't know where it would fit best, thank you so much